src/SFA.DAS.Campaign.UnitTests/Web/Controllers/Article/WhenRequestingTheArticlePage.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.Campaign.Domain.Content;
+using SFA.DAS.Campaign.Infrastructure.Api.Queries;
+using SFA.DAS.Campaign.Infrastructure.Api.Responses;
+using SFA.DAS.Campaign.Web.Controllers.Redesign;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.Campaign.UnitTests.Web.Controllers.Article
+{
+    public class WhenRequestingTheArticlePage
+    {
+        private const string HubName = "hub";
+        private const string SlugName = "slug";
+
+        [Test, RecursiveMoqAutoData]
+        public async Task AndGivenValidHubAndSlugThenThePageIsReturned(
+            GetArticleQueryResult<Domain.Content.Article> mediatorResult, [Frozen] Mock<IMediator> mockMediator,
+            [Greedy] ArticleController controller)
+        {
+            SetupMediator(mediatorResult, mockMediator);
+
+            var controllerResult = await InstantiateController<ViewResult>(controller);
```

Comment:
Can we make sure these tests are validating that the hub and slug values are being correctly passed to the mediator query